js/config/scoring.js

```diff
@@ -242,6 +274,8 @@ YoastSEO.AnalyzerScoring = function( i18n ) {
                     matcher: "matches",
                     min: 1,
                     score: 9,
+
+                    /* translators: %1$d expands to the amount of subheadings, %2$d to the amount of subheadings with the focus keyword */
```

Comment:
"subheadings with the focus keyword" 
I'm not entirely sure here. Should this be "with the focus keyword" or "containing the focus keyword"?